libs/shared/services/src/lib/packs.service.ts

```diff
@@ -281,31 +271,20 @@ export class PacksService {
       packsWithActiveBids.map((p) => [p.templateId, p])
     )
 
-    const filterPack = this.createPackFilterFn({
-      priceHigh,
-      priceLow,
-      status,
-      reserveMet,
-    })
-
-    const sortPack = this.createPackSortFn({
-      sortBy,
-      sortDirection,
-    })
-
     const assemblePack = this.createPublishedPackFn(
       packLookup,
       packWithActiveBidsLookup
     )
 
+    const postQueryFilters = this.createPackFilterFn({ status, reserveMet })
```

Comment:
`status` is a derived field populated in `toPackBase` by comparing `releasedAt` and `auctionUntil` dates.

I could make a query that makes filters following the logic in `toStatus` (within CMSCacheAdapter), but that'd involve me adding some filter complexity around handling `orWhere` queries vs `where` queries (all of the status queries would need to be `or` conditionals, and right now the query builder is all `and` conditionals)

Additionally, I'd be duplicating the logic in toStatus, which could have odd effects if the two behave slightly differently.

Ultimately I opted to just filter it post DB lookup for now because of these considerations

Reply:
`reserveMet` is comparing the `activeBid` against the pack price. `activeBid` is derived in the `getPacksWithActiveBids` mapping step of this function, it's not stored on the db. 

Again this is a very timely value so I can't cache the field in the DB.

We also don't know what the active bid is until doing subsequent lookups after querying the cache table

Thus, it too lives as a post DB lookup

Reply:
Okay I did a DB lookup after banging my head against a wall for a day and a half, because otherwise we'd be filtering down the singular pages results which isn't good